src/ring_theory/algebra_tower.lean

```diff
@@ -326,4 +327,33 @@ theorem is_basis.smul {ι : Type v₁} {b : ι → S} {κ : Type w₁} {c : κ 
 by rw [← set.range_smul_range, submodule.span_smul hb.2, ← submodule.restrict_scalars'_top R S A,
     submodule.restrict_scalars'_inj, hc.2]⟩
 
+theorem is_basis.smul_repr
+  {ι κ : Type*} {b : ι → S} {c : κ → A}
```

Comment:
Hmm, when I see `\kappa` in the context of algebra, I think of a residue field of a local ring, or a base field of a tower... So maybe it is safer to use `\iota'`?

Reply:
Sure. Should I also change this in the previous lines?

Reply:
Well κ is the next letter after ι...

Reply:
@kckennylau Sure... I know. But we've been trying to make variable names more "readable". So it's good to keep common use cases in mind.
@Vierkantor Yes, please change the other occurences as well.